apps/web/src/hooks/useRegisterNameCallback.ts

```diff
@@ -85,6 +99,35 @@ export function useRegisterNameCallback(
   const normalizedName = normalizeEnsDomainName(name);
   const isDiscounted = Boolean(discountKey && validationData);
 
+  const prepareReverseRecordSignature = useCallback(async () => {
+    if (!address) throw new Error('No address');
+    if (!publicClient) throw new Error('No public client');
```

Comment:
Is this check needed? does it fail without it?

Reply:
oh good catch, we actually don't need public client in this callback at all, I think this is vestigial from a prior approach to getting the L2RR address